setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as readme_file:
+    readme = readme_file.read()
+
+requirements = ["casbin==0.8.4", "psycopg2-binary==2.8.6", "black==20.8b1"]
+
+setup(
+    name="postgresql-watcher",
+    version="0.0.1",
+    author="hsluoyz",
+    author_email="hsluoyz@gmail.com",

```

Comment:
@hsluoyz could u publish it on PyPi through your credentials?

Reply:
In fact, currently this package not released.
You can release it, but plz add casbin to maintainer after that.

Reply:
Release it now is not a good idea. Since this package didn't complete yet. I will later working on this

Reply:
yeah, no problem. I will look into other issues.